src/styles/blocks/card.scss

```diff
@@ -0,0 +1,118 @@
+/* #region Card */
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+
+  box-sizing: border-box;
+  width: 1024px;
+  justify-content: space-between;
+  // width: 2 * $paddingX + 3 * $gapX + 4 *$card-width;
+  // gap: 46px 48px ;
+  margin: 0 auto;
+  padding: 50px $cards-paddingX;
+
+  &__card {
+    margin-right: 48px;
+    margin-bottom: 46px;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    &:nth-last-child(-n+4) {
+      margin-bottom: 0;
+    }
+  }
+}
```

Comment:
You can create a seperate file called cards.scss and put this block

Reply:
I agree, is done now :D